examples/api/test/material/banner/material_banner.0_test.dart

```diff
@@ -0,0 +1,31 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/material/banner/material_banner.0.dart'
+    as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('shows all elements', (WidgetTester tester) async {
```

Comment:
Consider capitalizing the first letter of test descriptions (same for the other tests below and your other opened PRs).